### DISCUSSION TITLE
Feature request(s) - Notes/Files in maps

### DISCUSSION THREAD

**Discussion by johnmcd88** [osmandapp/OsmAnd#18663] [Ideas]
Hi,

I use OsmAnd+ exclusively for mapping on my Android devices, and one of the features I use frequently is the A/V notes plugin to provide context for my various trips and activities. I was wondering if it would be possible to add, in addition to the current A/V notes,  options for simple text notes and/or files to be associated with points on a map. For example, if I download a PDF flyer for an event that's going to take place somewhere along my route I'd like to add that file to the appropriate point on the map. For notes I'd like to be able to enter just plain text or a URL that I could click on to bring up a page in a web browser. I've tried using markers for adding notes but it gets hard to distinguish between actual route markers and the ones I use for notes.

p.s. Thanks for all of the fantastic work that's been done to make OsmAnd the best mapping app available!

John

**Comment by mariush444** [comment]
You can add very long "description" to every favorities and track points.
Link to online doc can be added to.

**Reply by johnmcd88** [reply]
mariush,

That's what I've been doing, but it gets mixed up with my real favorites, markers, etc. - what would be ideal is a unique icon that appears as a document or note. In regards to adding documents, what I'd like to be able to do is link to a local document that gets brought into OsmAnd just like A/V notes so I can share them..

Thanks for the assist

John

**Reply by mariush444** [reply]
Aaah. OK.
I don't like icon for a/v notes because I don't know what I commented (hotel, lake, zoo ...) and I prefer unique icon for a kind of POI.
You know that there are quite many icon in osmand but maybe this info help you that you can use any icon from osm map that is shown by osmand.
Anyway, of course,  it is workarround and not solution for your needs.

**Comment by trymeouteh** [comment]
Would like to see a notes feature that allows you view all notes and see them as markers on the map.
